src/tests/admin/conferenceroomoutoforderplanning/OutOfOrderCreatedInCurrentTimeIsDisplayedInHomeNowTile.java

```diff
@@ -0,0 +1,66 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SettingsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC11: Verify that an Out Of Order created in the current time is displayed in NOW tile 
+ * in Tablet's HomePage
+ * @author Yesica Acha
+ *
+ */
+public class OutOfOrderCreatedInCurrentTimeIsDisplayedInHomeNowTile {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("OutOfOrderPlanning");
+	String roomName = testData.get(3).get("Room Name");
+	String title = testData.get(3).get("Title");
+		
+	@Test(groups = "ACCEPTANCE")
+	public void testOutOfOrderCreatedInCurrentTimeIsDisplayedInHomeNowTile() {
+		String startDate = testData.get(3).get("Start date");
+		String endDate = testData.get(3).get("End date");
+		String startTime = testData.get(3).get("Start time (minutes to add)");
+		String endTime = testData.get(3).get("End time (minutes to add)");
+		String description = testData.get(3).get("Description");
+		
+		//Out Of Order Creation in Admin
+		HomeAdminPage homeAdminPage = new HomeAdminPage(); 
+		RoomsPage conferenceRoom = homeAdminPage.clickConferenceRoomsLink();
+		RoomOutOfOrderPlanningPage outOfOrder = conferenceRoom
```

Comment:
In all of your test cases you call outOfOrderPage, but here is just outOfOrder. take care of this